.buildkite/pipeline.yml

```diff
@@ -172,42 +172,42 @@ steps:
       #   artifact_paths: "test/*"
 
       - label: "Moist earth with slab surface - default: monin gray no_sponge idealinsol freq_dt_cpl"
-        command: "julia --color=yes --project=experiments/AMIP/modular/ experiments/AMIP/modular/coupler_driver_modular.jl --enable_threading true --coupled true --surface_scheme monin_obukhov --moist equil --vert_diff true --rad gray --energy_check true --mode_name slabplanet --t_end 10days --dt_save_to_sol 3600secs --dt_cpl 200 --dt 200secs --mono_surface true --h_elem 4 --precip_model 0M --run_name default_mono --job_id default_mono"
+        command: "julia --color=yes --project=experiments/AMIP/modular/ experiments/AMIP/modular/coupler_driver_modular.jl --FLOAT_TYPE Float64 --enable_threading true --coupled true --surface_setup PrescribedSurface --moist equil --vert_diff true --rad gray --energy_check true --mode_name slabplanet --t_end 10days --dt_save_to_sol 3600secs --dt_cpl 200 --dt 200secs --mono_surface true --h_elem 4 --precip_model 0M --run_name default_mono --job_id default_mono"
```

Comment:
These are for atmos stand-alone runs only - they trigger the flux calculation during each atmos `step!` (and its substages), so we want to avoid that. I've commented on this in the driver and hoping that the terminology will become clearer in the next PR.